setup.py

```diff
@@ -8,6 +8,18 @@
 with open(path.join(cur_directory, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
+
+# get requirements from requirmeents.txt
+def get_reqs():
+    reqs=[]
+    with open("requirements.txt") as file:
+        for line in file:
+            # remove linebreak which is the last character of the string
+            currentReq = line[:-1]
+            # add item to the list
```

Comment:
i don't think you need this comment, i think folks know that `append` adds something to the list.  I would also remove the comment above this function as it feels a bit redundant with the name of the function.  The comment about removing the linebreak is a solid comment because a user might be wondering "why are we removing the last character in the line?"

Reply:
Ok sounds good I will make these changes now!